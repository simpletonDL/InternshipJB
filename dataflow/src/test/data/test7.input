i = 1
y = 1
while (i < 10) {
    y = i
    if (i < 5) {
        y = y + i
    }
    i = i + 1
}
x = y