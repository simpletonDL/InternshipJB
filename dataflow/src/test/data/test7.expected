Assignment x = y not used in line 10
